WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -236,6 +246,36 @@ public class BlockEditorScreen: ScreenObject {
         blockButton.tap()
     }
 
+    @discardableResult
+    public func undo() throws -> BlockEditorScreen {
+        XCTAssertTrue(undoButton.waitForIsHittable(timeout: 3))
+        undoButton.tap()
+
+        return try BlockEditorScreen()
+    }
+
+    @discardableResult
+    public func undoIsDisabled() throws -> BlockEditorScreen {
+        XCTAssertFalse(undoButton.isEnabled)
+
+        return try BlockEditorScreen()
+    }
+
+    @discardableResult
+    public func redo() throws -> BlockEditorScreen {
+        XCTAssertTrue(redoButton.waitForIsHittable(timeout: 3))
+        redoButton.tap()
+
+        return try BlockEditorScreen()
+    }
```

Comment:
Same as in `undo()`.

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20930/commits/ce5d63754b3a26091176e73640bea72e5720326c